stories/ConsentManagement.stories.tsx

```diff
@@ -121,7 +122,13 @@ export const {
         }
         */
 
-    }
+    },
+
+    /*
+    This optional parameter let's you personalise the key that is used to store user's consents in the localStorage.
+    The default value is "${defaultLocalStorageKeyPrefix}"
+    */
+    "localStorageKeyPrefix": "company-name/app-name"
```

Comment:
I'd like to keep this undocumented.  
It's a niche thing that is relevent in your usecase but for most app it won't be.  
Pepole don't read documentation. If they see the doc is a little bit too long they'd rather implement their own solution.  
So let's keep only the essential info.

Reply:
Ok, I removed all mentions in the documentation.